### DISCUSSION TITLE
After the donation, the speed is even slower.

### DISCUSSION THREAD

**Discussion by Simon1987wee** [iterate-ch/cyberduck#14733] [Q&A]
Why upload becomes very slow? It can reach 5++kb/s without using cyber duck upload.
After using cyberduck it is even worse, only 28kb/s.

**Comment by dkocher** [comment]
Having the application registered has no side effects whatsoever to application performance or transfer throughput.

**Reply by Simon1987wee** [reply]
So what's the problem?

**Reply by dkocher** [reply]
This must be either a problem with your Internet connection or the server from the hosting service provider.

**Reply by Simon1987wee** [reply]
Fortunately, you still have a product (mountain Duck). I tried the function, and the upload speed can reach 65mb/s.

**Comment by Simon1987wee** [comment]
Cyber ​​Duck was really good when I first started using it.But now I'm
mostly useless, because I'm very disappointed with the upload speed! !

David Kocher ***@***.***> 于 2023年5月31日周三 下午4:56写道：

**Comment by Simon1987wee** [comment]
Hello, can you let me know what is the status of cyberduck's current
program?Or is this cyberduck already a joke program.
I personally think that using this cyberduck upload is already a whimsical
thing. .

Simon Chieu ***@***.***> 于2023年6月13日周二 18:06写道：
